### PR TITLE
Toolbars v3 overflow

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -2,11 +2,17 @@ import React, { Component } from 'react';
...
+import { withToolbarContext } from '../../utils/withContext';
+import { OverflowButton } from '../buttons';
+import { debounce } from 'lodash';
+import type { ToolbarContextType } from '../../utils/toolbarContexts';
 
 type ToolbarProps = {
   toolbar: RicosToolbar;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   toolbarItemsRenders: any;
+  context: ToolbarContextType;
```
Comment (justame) [inline packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx:15]: why toolbar should get context as a prop? let's talk about it

File: packages/toolbars-v3/web/src/components/buttons/OverflowButton/OverflowButton.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    background-color: #ffffff;
+    display: flex;
+    // position: absolute;
+    // top: 36px;
+    z-index: 10;
+    width: max-content;
+    min-width: 42px;
+    @include border-box;
+    @include phone {
+      &:not(.colorPickerMobile) {
```
Comment (justame) [inline packages/toolbars-v3/web/src/components/buttons/OverflowButton/OverflowButton.scss:44]: why a generic component has a reference to colorPickerMobile?

Comment (noamtwix) [reply packages/toolbars-v3/web/src/components/buttons/OverflowButton/OverflowButton.scss:44]: you are right.
These are leftovers from the code I took.
cleaning it up.
thanks

File: packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx [hunk 3]
```diff
@@ -28,20 +32,51 @@ class ToolbarComponent extends Component<ToolbarProps, Record<string, unknown>>
...
+    window.addEventListener('resize', this.onWindowResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.onWindowResize);
+  }
+
+  onWindowResize = debounce(() => {
+    // force update
+    this.setState({ dummyUpdate: this.state.dummyUpdate + 1 });
```
Comment (justame) [inline packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx:45]: why the need to force an update?

Comment (noamtwix) [reply packages/toolbars-v3/web/src/components/Toolbar/Toolbar.tsx]: to call the `getToolbarWidth` function again.
I change it..
I think it's better now.
what do you think?

**Pull request closed without merging**